src/utilities/new_versions.jl

```diff
@@ -340,21 +342,51 @@ function unsub_from_pr(api::GitLab.GitLabAPI, pr::GitLab.MergeRequest)
 end
 
 function add_compat_entry(
-    name::AbstractString, repo_path::AbstractString, brand_new_compat::AbstractString
+    name::AbstractString,
+    repo_path::AbstractString,
+    brand_new_compat::AbstractString;
+    bump_version::Bool=false,
 )
     project_file = joinpath(repo_path, "Project.toml")
     project = TOML.parsefile(project_file)
 
     add_compat_section!(project)
     project["compat"][name] = brand_new_compat
 
+    bump_version && bump_package_version!(project)
+
     open(project_file, "w") do io
         TOML.print(
             io, project; sorted=true, by=key -> (Pkg.Types.project_key_order(key), key)
         )
     end
 end
 
+function bump_package_version!(project::Dict)
+    version = VersionNumber(project["version"])
+
+    # Only bump the version if prerelease is empty
+    !isempty(version.prerelease) && return nothing
+
+    # Bump minor if version > 1.0, else bump patch
+    if version.major >= 1
+        version = VersionNumber(
+            version.major, version.minor + 1, 0, version.prerelease, version.build
```

Comment:
I don't think prerelease or build are used in compat entries, so they're not needed?
I feel like keeping the same build number especially is wrong, as that would not really exist?

Reply:
This updates the package version, not the compat version. That being said, maybe this shouldn't be in the same function that updates the compat version